lib/util/http/github.ts

```diff
@@ -288,7 +314,7 @@ export class GithubHttp extends Http<GithubHttpOptions, GithubHttpOptions> {
 
     try {
       const result = await super.request<T>(url, opts);
-      if (opts.paginate) {
+      if (options?.paginate) {
```

Comment:
Why this change?

Reply:
🤷‍♂️ Doesn't matter, will revert 🙃 
```suggestion
      if (opts?.paginate) {
```